### ISSUE TITLE
[AE2] Import buses and molecular assemblers bug.

### ISSUE THREAD

**Issue by defaultunknown** [DarkPacks/SkyFactory-4#2154]
## Bug Description

all import buses stopping working in ae2 system when more than 5 molecular assemblers work at the same time with 1 import bus, 1 export bus on said assembler and recipe on storage pattern is present inside of each molecular assembler.

*so called true autocrafting without requesting resource/crafting every time.

**If required, I can upload world with this issue.

### Did this behavior use to work in the previous version?

I don't know if in the previous versions this bug was present.

## Steps to Reproduce (for bugs)

1. create ME network with enough power for it to feed and enough storage (e.g. 64k storage cell) to operate.
2. add 16±2 molecular assemblers, hook import/export buses and power up them.
3. provide resources for _infinite_ crafting with assemblers (e.g. acorn + resin -> amber, from import buses hooked to bonsai pots).
4. observe this ~~feature~~ bug after <1 minute of autocrafting.

## Logs

* Client/Server Log: Not required, doesn't log
* Crash Log: None

## World Information
* World Preset:{"Topography-Preset":"Sky Factory 4"}
* Prestige Enabled: False
* Modpack Version world created in: version 4.2.4
* Additional Content Installed: OfflineSkins 1.12.2 v6

## Client Information
* Modpack Version: 4.2.4
* Java Version: jre1.8.0_341
* Launcher Used: MultiMC 0.6.16-3431 win32
* Memory Allocated: 8GB
* Server/LAN/Single Player: Server
* Optifine Installed: yes
* Shaders Enabled: no

## Server Information
* Java Version: jdk1.8.0_333
* Operating System: Windows 11 Pro, 21H2 (OS Build 22000.856)
* Hoster/Hosting Solution: Local machine (server) exposed to wan
* Sponge (Non-Vanilla Forge) Server: No

**Comment by sam-kirby** [comment]
Do you have enough energy storage on the AE2 side?

The energy acceptor converts RF to AE2's own energy system, and it can only accept as much RF as the network has storage for per tick.

If your use is higher per tick than you have AE2 storage for, then you will experience devices switching off.

If you don't have any energy cells or dense energy cells then add some, as it sounds like you're running into an energy bottleneck.

You can evaluate this in more detail using AE2's smart wrench.

**Comment by defaultunknown** [comment]
thank you for reply

1) yes, I do have enough energy: 7 dense energy cells, 2 energy acceptors to feed them.
2) see 1
3) all the devices were "online" whenever I was testing this bug
4) see 1
5) I never heard about "smart wrench", if you mean "network tool" then I did check everything with it

5.1) If being ultra precise, the bug was observed with 16 assemblers, 16 import buses, 16 export buses, 2 dense energy cells, 2 energy acceptors, creative energy cube (mekanism), 16 pattern storage with random crafting table recipes (mostly acorns and/or resin), 720 acceleration cards (16*4*2+16*5), me drive with 4x64k drives filled at the moment nearly 47% of all drives, 26 import buses on 26 bonsai pots with blue mulch and various of trees producing acorns and/or resin with 4*26 acceleration card and configured to not output leaves/wood/sticks.

P.S. I found temporary workaround with seconds me system with omnidirectional providing of acorns but still want everything in one system.

**Comment by sam-kirby** [comment]
You're right, I meant the network tool but I didn't have the pack open to check the name.

I've made some pretty expansive ME systems in this pack and not run into this issue, so could you share your world somehow and I'll see if I can figure out if this is a bug and if so what exactly the cause is.

**Comment by defaultunknown** [comment]
Sure, I will upload it to my web server, hope it's okay.
The assemblers are at -1 floor (basically under the island)

[https://defaultsant.com/public_data/Backup--world--2022-09-03--22-14.zip](https://defaultsant.com/public_data/Backup--world--2022-09-03--22-14.zip)

**Comment by sam-kirby** [comment]
> Sure, I will upload it to my web server, hope it's okay. The assemblers are at -1 floor (basically under the island)
> 
> https://defaultsant.com/public_data/Backup--world--2022-09-03--22-14.zip

503 error.

**Comment by defaultunknown** [comment]
weird. okay, gonna upload to onedrive then:
[https://1drv.ms/u/s!AohlFy8W7t1FgQK1we7ymT1oO72w?e=MXVTk0](https://1drv.ms/u/s!AohlFy8W7t1FgQK1we7ymT1oO72w?e=MXVTk0)
